Tasks/DotNetCoreCLIV2/packcommand.ts

```diff
@@ -138,7 +138,12 @@ function dotnetPackAsync(dotnetPath: string, packageFile: string, outputDir: str
     let dotnet = tl.tool(dotnetPath);
 
     dotnet.arg("pack");
-    dotnet.arg(packageFile);
+
+    if(packageFile.endsWith(".nuspec"))
+    {
+        dotnet.arg("-p:NugetConfig="+packageFile);
```

Comment:
should it be `NuspecFile` instead of `NugetConfig` ? 
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack#examples

Reply:
yes updated now, was trying with `--p` flag earlier thus switched to `NugetConfig` option instead.